src/command.rs

```diff
@@ -107,6 +139,18 @@ fn publish(path: Option<String>) -> result::Result<(), Error> {
     Ok(())
 }
 
+fn login(
+    registry: Option<String>,
+    scope: Option<String>,
+    always_auth: bool,
+    auth_type: Option<String>,
+) -> result::Result<(), Error> {
+    npm::npm_login(registry, scope, always_auth, auth_type)?;
+
+    PBAR.message("👋  logged you in!");
```

Comment:
for this, i think it'd be good to print what registry they were logged into :) (whatever was provided, or the default registry.npmjs.com)

Reply:
Unless we somehow want to filter and or parse the npm output…
For now i'd suggest keeping the wasm-pack output as concise as possible (without registry info). What do you think?

Reply:
*gentlypush* :)

Reply:
i didn't realize it was redundant! no worries, feel free to not print it here! my main concern was that it was printed at all :)